ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextShadowNode.java

```diff
@@ -418,6 +422,15 @@ public void setLineHeight(int lineHeight) {
     markUpdated();
   }
 
+  @ReactProp(name = ViewProps.ALLOW_FONT_SCALING)
+  public void setAllowFontScaling(@Nullable Boolean allowFontScaling) {
+    mAllowFontScaling = allowFontScaling != null && allowFontScaling.booleanValue();
```

Comment:
Do these scenarios work correctly?

1. Developer sets `allowFontScaling` to `null`. It looks to me like this will cause `mAllowFontScaling` to be `false` but the default should be `true`. To fix this, remove the `@Nullable` annotation and use `@ReactProp`'s `defaultBoolean = true` feature. I think that'll make the code clearer and correct.

2. Developer toggles the value of `allowFontScaling` when `mFontSizeInput` is `UNSET`. I suspect the text will not be rerendered to account for the change in the `allowFontScaling` value because `markUpdated` is never called.

3. Developer sets `allowFontScaling` to `false` and then to `true`. I suspect the text will not be rerendered to account for the change in the `allowFontScaling` value because `markUpdated` is never called.